Gemfile

```
@@ -352,3 +352,5 @@ gem 'cld'
 gem 'crowdin-api', '~> 1.6.0'
 
 gem "delayed_job_active_record", "~> 4.1"
+
+gem 'rack-cors'
```

nit; I'd like to at least vaguely target a version we know works with our environment, for future-proofing. Something like,
```suggestion
gem 'rack-cors', '~> 2.0'
```
or even,
```suggestion
gem 'rack-cors', '~> 2.0.1'
```

---

Good call! Took your more specific suggestion